### PR TITLE
radio-item component

### PR DESCRIPTION
**Pull request by anveshmekala**
Create radio-item component to slot the radio-buttons with radio-group component

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -9,10 +9,9 @@
     <script nomodule src="/build/stencil-starter-project-name.js"></script>
   </head>
   <body>
-    <anveshmekala-radio-group></anveshmekala-radio-group>
-    <script>
-      const radioGroup = document.querySelector('anveshmekala-radio-group')
-      radioGroup.list = ['batman','thor','loki']
-      </script>
+    <anveshmekala-radio-group>
+      <radio-item label="Batman" ></radio-item>
```
Comment (driskull) [inline src/index.html]: seems like this should have the same prefix `anveshmekala ` as the parent.

File: src/components/radio-group/test/radio-group.spec.tsx [hunk 2]
```diff
@@ -1,12 +1,18 @@
...
-   expect(radioGroup.selectedIndex).toBe(2)
-   expect(radioGroup.selectedItem).toBe("SpiderMan")
- })
+    it('renders', async () => {
+        const page = await newSpecPage({
+          components: [RadioGroup],
+          html: `<radio-group></radio-group>`,
+        });
+        expect(page.root).toEqualHtml(`
+          <radio-group>
```
Comment (driskull) [inline src/components/radio-group/test/radio-group.spec.tsx]: maybe some radio-items should be put in here. Also doesn't the component need the prefix `anveshmekala`?

**Review by driskull — COMMENTED** [review]
Looks awesome! Just noticed a couple things.

File: src/components/radio-group/test/radio-group.spec.tsx (hunk 2 above)
Comment (anveshmekala) [reply src/components/radio-group/test/radio-group.spec.tsx]: updated in [97cb909](https://github.com/anveshmekala/stencil-sandbox/pull/6#:~:text=update%20unit%20test-,97cb909,-Merge%20state)

**Pull request merged**